ext/ddtrace.stub.php

```diff
@@ -0,0 +1,496 @@
+         * @var string The service you are tracing
+         */
+        public string $service;
+
+        /**
+         * @var string The type of request which can be set to: web, db, cache, or custom (Optional)
+         */
+        public string $type;
+
+        /**
+         * @var string[] $meta An array of key-value span metadata; keys and values must be strings (Optional)
+         */
+        public array $meta;
+
+        /**
+         * @var float[] $metrics An array of key-value span metrics; keys must be strings and values must be floats
+         * (Optional)
+         */
+        public array $metrics;
+
+        /**
+         * @var \Throwable $exception An exception generated during the execution of the original function, if any.
+         */
+        public \Throwable $exception;
+
+        /**
+         * @var string The unique identifier of the span
+         */
+        public string $id;
+
+        /**
+         * @var SpanData|null The parent span, or 'null' if there is none
+         */
+        public SpanData|null $parent;
+
+        /**
+         * @var SpanStack The span's stack trace
+         */
+        public SpanStack $stack;
+    }
+
+    class SpanStack {
+        /**
+         * @var SpanStack|null The parent stack, or 'null' if there is none
+         */
+        public SpanStack|null $parent;
+
+        /**
+         * @var SpanData The active span
+         */
+        public SpanData $active;
+    }
+
+    /**
+     * Close child spans of a parent span if a non-internal span is given,
+     * else if 'null' is given, close active, non-internal spans
+     * @param SpanData|null $span The parent span
+     * @return bool|int 'false' if spans couldn't be closed, else the number of span closed
+     */
+    function close_spans_until(?SpanData $span): bool|int {}
+
+    /**
+     * Get the active span
+     * @return SpanData|null 'null' if tracing isn't enabled or there isn't any active span, else the active span
+     */
+    function active_span(): null|SpanData {}
+
+    /**
+     * Get the root span
+     * @return SpanData|null 'null' if tracing isn't enabled or if the active stack doesn't have a root span,
+     * else the root span of the active stack
+     */
+    function root_span(): null|SpanData {}
+
+    /**
+     * Start a new custom user-span on the top of the stack
+     * @param \DateTimeInterface|int|float $startTime Start time of the span in seconds. Defaults to zero.
+     * @return SpanData The newly started span
+     */
+    function start_span(\DateTimeInterface|int|float $startTime = 0): SpanData {}
+
+    /**
+     * Close the currently active user-span on the top of the stack
+     * @param \DateTimeInterface|int|float|null $finishTime Finish time in seconds. if passing float or int, it should
+     * represent the timestamp (including as many decimal places as you need)
+     * @return void
+     */
+    function close_span(\DateTimeInterface|int|float|null $finishTime): void {}
```

Comment:
```suggestion
    function close_span(float $finishTime = 0): void {}
```
Same than start_span